src/main/java/it/gov/pagopa/ranking/dto/RankingPageDTO.java

```diff
@@ -23,10 +23,17 @@ public class RankingPageDTO {
     private RankingStatus rankingStatus;
     private LocalDateTime rankingPublishedTimeStamp;
     private LocalDateTime rankingGeneratedTimeStamp;
+
+    // Number of eligible users
```

Comment:
use javadoc notation when documenting 
```suggestion
    /** Number of eligible users */
```